source/dem/velocity_verlet_integrator.cc

```diff
@@ -130,5 +130,123 @@ VelocityVerletIntegrator<dim>::integrate(
     }
 }
 
+template <int dim>
+void
+VelocityVerletIntegrator<dim>::integrate(
+  Particles::ParticleHandler<dim> &                particle_handler,
+  const Tensor<1, 3> &                             g,
+  const double                                     dt,
+  std::vector<Tensor<1, 3>> &                      torque,
+  std::vector<Tensor<1, 3>> &                      force,
+  const std::vector<double> &                      MOI,
+  const parallel::distributed::Triangulation<dim> &triangulation,
+  typename DEM::dem_data_structures<dim>::cell_index_int_map
+    &cell_mobility_status_map)
+{
+  Point<3>           particle_position;
+  const Tensor<1, 3> dt_g = g * dt;
+
+  for (auto &cell : triangulation.active_cell_iterators())
+    {
+      if (cell->is_locally_owned())
+        {
+          // Check if the cell is mobile, if not, integration is skipped
+          unsigned int mobile_status  = 2; // Value of the mobile status
```

Comment:
why not use the enum directly to set the value of the mobile status? This seems dangerous